go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/upbound/provider-gcp
 
-go 1.19
```

Comment:
We will also need to update the go versions in the CI definitions.

Reply:
Thanks @ulucinar for the comment. I updated uptest workflows as well.